pkg/reconciler/ingress/config/istio_test.go

```diff
@@ -253,3 +253,61 @@ func TestGatewayConfiguration(t *testing.T) {
 		})
 	}
 }
+
+func TestVSStatusEnabled(t *testing.T) {
+	vsStatusTests := []struct {
+		name        string
+		wantErr     bool
+		wantEnabled interface{}
+		config      *corev1.ConfigMap
+	}{{
+		name:        "enabled",
+		wantEnabled: true,
+		config: &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: system.Namespace(),
+				Name:      IstioConfigName,
+			},
+			Data: map[string]string{
+				"enable-virtualservice-status": "true",
```

Comment:
Could you add const value for `enable-virtualservice-status` in `./pkg/reconciler/ingress/config/istio.go` and use it here?

Reply:
I've added a const in `istio.go`, but I don't think we should use it here. This way if someone accidentally changes the const in `istio.go` the test will fail.